app/views/hyrax/base/_form.html.erb

```diff
@@ -1,31 +1,36 @@
-<%= simple_form_for [main_app, @form],
-                    html: {
-                      data: { behavior: 'work-form',
-                              'param-key' => @form.model_name.param_key },
-                      multipart: true
-                    } do |f| %>
-  <% if f.object.errors.any? %>
-    <div class="alert alert-danger alert-dismissable" role="alert">
-      <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-      <%= f.object.errors.full_messages_for(:base).send(SimpleForm.error_method) %>
-      <%= render 'form_in_works_error', f: f %>
-      <%= render 'form_ordered_members_error', f: f %>
-      <%= render 'form_collections_error', f: f %>
-      <%= render 'form_visibility_error', f: f %>
-    </div>
-  <% end %>
-  <% if Flipflop.batch_upload? && f.object.new_record? %>
-    <% provide :metadata_tab do %>
-      <p class="switch-upload-type">To create a separate work for each of the files, go to <%= link_to "Batch upload", hyrax.new_batch_upload_path %></p>
+<div class="simple_works_form">
+  <%= simple_form_for [main_app, @form],
+                      html: {
+                        data: { behavior: 'work-form',
+                                'param-key' => @form.model_name.param_key },
+                        multipart: true
+                      } do |f| %>
+    <% if f.object.errors.any? %>
+      <div class="alert alert-danger alert-dismissable" role="alert">
+        <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <%= f.object.errors.full_messages_for(:base).send(SimpleForm.error_method) %>
+        <%= render 'form_in_works_error', f: f %>
+        <%= render 'form_ordered_members_error', f: f %>
+        <%= render 'form_collections_error', f: f %>
+        <%= render 'form_visibility_error', f: f %>
+      </div>
+    <% end %>
+    <% if Flipflop.batch_upload? && f.object.new_record? %>
+      <% provide :metadata_tab do %>
+        <p class="switch-upload-type">To create a separate work for each of the files, go to <%= link_to "Batch upload", hyrax.new_batch_upload_path %></p>
+      <% end %>
     <% end %>
+    <%= render 'hyrax/base/guts4form', f: f %>
   <% end %>
-  <%= render 'hyrax/base/guts4form', f: f %>
-<% end %>
+</div>
 
 <script type="text/javascript">
   Blacklight.onLoad(function() {
     <%# This causes the page to switch back to the default template if they've
         previously visited the batch download page in this Turbolinks session %>
     $("#fileupload").fileupload('option', 'downloadTemplateId', 'template-download')
+    $('.simple_works_form').on('submit', function(event) {
+      $('.panel-footer').toggleClass('hidden');
```

Comment:
This should be scoped to the `.panel-footer` that is within the `data-behavior='work-form'`